commit 954469da6d4c265e194b2442e8ee58f9d8317c53
Author: Phat Huynh <sleepyorion@gmail.com>
Date:   Wed Jul 5 08:15:44 2017 -0700

    Modified existing function to override attribute name to be lowercase.

diff --git a/lib/ridley/chef/cookbook/metadata.rb b/lib/ridley/chef/cookbook/metadata.rb
index 02884f2..fc541b8 100644
--- a/lib/ridley/chef/cookbook/metadata.rb
+++ b/lib/ridley/chef/cookbook/metadata.rb
@@ -245,6 +245,7 @@ module Ridley::Chef
       # === Returns
       # name<String>:: Returns the current cookbook name.
       def name(arg = nil)
+        arg = arg.nil? ? nil : arg.downcase
         set_or_return(
           :name,
           arg,
